### DISCUSSION TITLE
点击播放视频闪退

### DISCUSSION THREAD

**Discussion by TaylorSooong** [Richasy/Bili.Uwp#501] [问答]
打开应用后，随便点开一个视频就闪退，程序日志都正常。
![image](https://user-images.githubusercontent.com/84616153/137581991-8ca365b0-99d5-4b3d-9dac-064ac8d96ea2.png)

**Comment by TaylorSooong** [comment]
@Richasy

**Reply by ClewenWei** [reply]
我之前也有这个问题，在可靠性历史记录里和一个 dll 文件有关，重装显卡驱动也没有解决。并且同一个开发者的 RSS 追踪也有类似的情况。
不过现在重装系统之后没问题了。
